packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroup.Props.ts

```diff
@@ -63,6 +63,11 @@ export interface IChoiceGroupOption {
   onRenderField?: IRenderFunction<IChoiceGroupOption>;
 
   /**
+   * Optional override of option render
+   */
+  onRenderLabel?: (IChoiceGroupOption) => JSX.Element;
```

Comment:
I initially did that, but as a render function in a render function, I had to insert the onRenderLabel into the option at the render function. Otherwise when I did `onRenderLabel(option, this._onRenderLabel)` it would fail as it didn't have access to `this`

![image](https://user-images.githubusercontent.com/1434956/27931646-be27d0d6-624f-11e7-853c-2b2b42d3be2a.png)

I'd be happy to be wrong though!